### PR TITLE
Docs: correct options for possible location of `rose.conf`

### PR DESCRIPTION
**Pull request by jmarshrossney**
Changed 2 lines in [the docs](https://metomi.github.io/rose/doc/html/installation.html#configuring-rose) for configuring rose:

`/etc/.metomi/rose.conf` --> `/etc/rose.conf` \
`$ROSE_SITE_CONF_PATH/.metomi/rose.conf` --> `$ROSE_SITE_CONF_PATH/rose.conf`

The [docstring](https://metomi.github.io/rose/doc/html/api/configuration/site-and-user.html#rose:file:rose.conf) is correct.

Cheers!

### CODE REVIEW

**Review by oliver-sanders — APPROVED** [review]
Took a look at the code, this all seems correct to me.

Thanks!

**Comment by oliver-sanders** [comment]
Thanks for sorting this out @jmarshrossney.

For legal reasons, when people make their first contribution to Rose (no matter how big or small), they need to add their name to the [`CONTRIBUTING.md`](https://github.com/metomi/rose/blob/master/CONTRIBUTING.md) file before we are allowed to accept their commit. This indicates that you have accepted the contributor licence agreement (CLA).

Could you please add `marshrossney` to the `CONTRIBUTING.md`](https://github.com/metomi/rose/blob/master/CONTRIBUTING.md) file. Note, this must match the format returned by `git shortlog -se`.

If you would like to change the appearance of your name, then add an entry to the `.mailmap` file like so:

```diff
diff --git a/.mailmap b/.mailmap
index f1e6d5759..9b4a4794a 100644
--- a/.mailmap
+++ b/.mailmap
@@ -35,3 +35,4 @@ Thomas Coleman       <thomas.coleman@bom.gov.au>          Tom Coleman          <
 Tim Pillinger        <tim.pillinger@metoffice.gov.uk>     wxtim                <tim.pillinger@metoffice.gov.uk>
 Tim Pillinger        <26465611+wxtim@users.noreply.github.com>
 Tomasz Trzeciak      <tomasz.trzeciak@metoffice.gov.uk>   TomekTrzeciak        <tomasz.trzeciak@metoffice.gov.uk>
+First Last           <17361029+marshrossney@users.noreply.github.com>

diff --git a/CONTRIBUTING.md b/CONTRIBUTING.md
index 541e8889e..1f819233e 100644
--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -53,6 +53,7 @@ below:
  - Ronnie Dutta (Met Office, UK)
  - Roddy Sharp (Met Office UK)
  - Mark Dawson (Met Office UK)
+ - First Last
 <!-- end-shortlog -->
 
 (All contributors are identifiable with email addresses in the version control
```

**Comment by jmarshrossney** [comment]
Fair dos

File: .mailmap [hunk 1]
```diff
@@ -35,3 +35,4 @@ Thomas Coleman       <thomas.coleman@bom.gov.au>          Tom Coleman          <
 Tim Pillinger        <tim.pillinger@metoffice.gov.uk>     wxtim                <tim.pillinger@metoffice.gov.uk>
 Tim Pillinger        <26465611+wxtim@users.noreply.github.com>
 Tomasz Trzeciak      <tomasz.trzeciak@metoffice.gov.uk>   TomekTrzeciak        <tomasz.trzeciak@metoffice.gov.uk>
+Joe Marsh Rossney    <17361029+marshrossney@users.noreply.github.com>
```
Comment (oliver-sanders) [inline .mailmap:38]:
```suggestion
Joe Marsh Rossney    <17361029+jmarshrossney@users.noreply.github.com>   marshrossney    <17361029+marshrossney@users.noreply.github.com>
Joe Marsh Rossney    <17361029+jmarshrossney@users.noreply.github.com>   jmarshrossney    <17361029+jmarshrossney@users.noreply.github.com>
```

**Comment by jmarshrossney** [comment]
Apologies for the github username/email complications! That's my mistake.

**Comment by oliver-sanders** [comment]
No probs, it's just a hoop we have to jump through.

**Pull request merged**